playbooks/inventory.ini

```diff
@@ -0,0 +1,2 @@
+[collector]
+ec2 ansible_host=ec2-52-91-127-75.compute-1.amazonaws.com ansible_user=ec2-user ansible_private_key_file=~/.ssh/id_rsa
```

Comment:
Use the elastic IP address instead.

Reply:
I used the DNS because after stopping and restarting the VM it's IP had changed.
Isn't it problematic?

Reply:
> I used the DNS because after stopping and restarting the VM it's IP had changed. Isn't it problematic?

Elastic IP is a different concept, different from private or public IP of EC2. More details [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html).

Reply:
Let's sync up about this in the meeting.